tests/in/glsl/sampler-functions.frag

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Could you rerun tests with the `wgsl-out` feature? It looks like your PR missing wgsl output snapshots.